test/test-cases/scale/vnet2vnet/README.md

```diff
@@ -95,6 +95,7 @@ The files found within this directory serve the following purpose:
 
 # test cases
 
-| Test case                                      | Description                                               |
-| ---------------------------------------------- | --------------------------------------------------------- |
-| [vxlan_1vpc_1ip](README.vxlan_1vpc_1ip.md)     | performance numbers for best case scenario                |
+| Test case                                      | Description                                                    |
+| ---------------------------------------------- | -------------------------------------------------------------- |
+| [vxlan_1vpc_1ip](one-ip/README.md)             | performance numbers for best case scenario                     |
+| [vxlan_8vpc_48K-ips](48K-ips/README.md)        | performance with an objective of maintaining 6M parallel flows |

```

Comment:
This link appears to be broken although it "looks" correct.

Reply:
This one looks good, I can follow it. But the same one on the line 15 I will fix it.